app/components/certificate/certificate-display.tsx

```diff
@@ -0,0 +1,100 @@
+import {
+  Flex,
+  Box,
+  HStack,
+  Text,
+  Heading,
+  Tooltip,
+  IconButton,
+  Accordion,
+  AccordionItem,
+  AccordionButton,
+  AccordionIcon,
+  AccordionPanel,
+  useToast,
+} from '@chakra-ui/react';
+import { DownloadIcon, CopyIcon } from '@chakra-ui/icons';
+
+interface CertificateDisplayProps {
+  title: string;
+  value: string;
+  description: string;
+}
+
+export default function CertificateDisplay(props: CertificateDisplayProps) {
+  const { title, description, value } = props;
```

Comment:
nit: 

```ts
export default function CertificateDisplay({ title, description, value }: CertificateDisplayProps) {
```

Reply:
oh that makes it much better than calling it again inside